File: Jarvis.Web/MG.Jarvis.Web/ClientApp/src/app/extranet/viewmodel/associatemealplanviewmodel/mealplanviewmodel.ts
import { MealOptionViewModel } from './mealoptionviewmodel';
import { IObjectWithState } from '../../../common/iobjectwithstate';
import { ObjectState } from '../../../common/enums';

export class MealPlanViewModel  implements IObjectWithState {
    objectState: ObjectState;
    hotelId: number;
    mealId: number;
    mealName: string;
    isSelected: boolean;
    mealPlanOptions: MealOptionViewModel;
}

File: Jarvis.Web/MG.Jarvis.Api.Extranet/ViewModel/MealPlanViewModel.cs
namespace MG.Jarvis.Api.Extranet.ViewModel
{
    public class MealPlanViewModel : BaseViewModel
    {
        public int MealId { get; set; }
        public bool IsSelected { get; set; }
        public MealOptionViewModel MealPlanOptions { get; set; }
    }
}
